.kitchen.vagrant.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
you shouldn't need this? `kitchen.yml` should provide this.

Reply:
Maybe not :)
You can aggregate dokken and vagrant config as the same time ?